k8s/jiva/unblock_service.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+usage()
+{
+    echo "Usage: bash script.sh <controller_pod_name> <pvc_namespace>"
```

Comment:
Same comment like in previous script. Also it is possible to club this into the previous script itself, to avoid code duplication.